WcaOnRails/app/views/static_pages/contact.html.erb

```diff
@@ -1,49 +1,45 @@
-<% provide(:title, 'Contact Information') %>
+<% provide(:title, t('contact.title')) %>
 
 <div class="container">
   <h1><%= yield(:title) %></h1>
 
   <%= alert :info do %>
-    Before contacting any of the teams on this page,
-    please take a look at our <%= link_to "frequently asked questions", faq_path %>!
+    <%= raw t 'contact.info_alert', link: link_to(t('layouts.navigation.faq'), faq_path) %>
   <% end %>
 
   <p>
-    For website questions, comments, and suggestions, use this
-    <a href="/contact/website">contact form</a>. For inquiries concerning a
-    specific area or competition, please contact your local <%= link_to "WCA
-    Delegate", delegates_path %>.
+    <%= raw t 'contact.questions_and_comments', link: link_to("WCA Delegate", delegates_path) %>
```

Comment:
We have the key [`competitions.competition_info.delegate`](https://github.com/thewca/worldcubeassociation.org/blob/277241c4010128ae420e36e3b5948b13b38528e2/WcaOnRails/config/locales/en.yml#L620) if needed.